README.md

```diff
@@ -910,6 +910,77 @@ end
 
 You can remember the method name by the convention that `remove` or `apply` describes it's function, `behavior` refers to the first argument (thet contsant holding the behaviors), and then the name of the role which refers to the role playing object: `remove_behavior_role`.
 
+##Name collisions between methods and roles
+Lets say that you wish to create a context as below, intending to use instances of the following two classes as actors:
+
+```ruby
+  class Postcode
+    # other methods...
+    def code
+      @code
+    end
+
+    def country
+      @country
+    end
+  end
+
+  class Country
+    # other methods...
+    def country_code
+      @code
+    end
+  end
+
+  class SendAParcel
+    extend Surrounded::Context
+
+    keyword_initialize :postcode, :country
+
+    trigger :send do
+      postcode.send
+    end
+
+    end
+    role :postcode do
+      def send
+        # do things...
+        country_code = country.country_code # raises an exception!
+      end
+    end
+  end
```

Comment:
This is supposed to detail what happens without name collision detection, perhaps the comment isn't particularly clear - I could change it to "name collision raises an exception" or similar.